src/content/blog/year-in-review-2024.mdx

```diff
@@ -0,0 +1,716 @@
+
+In this post, I'll summarize what I see as the most significant happenings in the Astro ecosystem in 2024. I'll also touch on what might be coming in 2025, as far as I'm aware.
+
+If you're interested, I've also published a [2023 edition (in Japanese)](https://zenn.dev/morinokami/articles/astro-2023-2024) of this article. Feel free to check it out for a look back at Astro's progress in 2023.
+
+
+## Project health indicators
+
+### Release Versions
+
+As of the start of 2024, Astro was at [v4.0.8](https://github.com/withastro/astro/releases/tag/astro%404.0.8), and by the time of writing this article near the end of the year, [v5.1.1](https://github.com/withastro/astro/releases/tag/astro%405.1.1) has been released. Including beta versions, **there were a total of 169 releases in 2024**. That's almost on par with the number of releases in 2023, which was 175.
+
+In 2023, the major version jumped from 1 to 4, giving the impression of a rapid pace of feature additions and improvements. In 2024, the major version only went from 4 to 5, and the pace of version bumps seems to have eased somewhat.
+
+For those who are curious about the details of each release, you can check [the full list of Astro releases on GitHub](https://github.com/withastro/astro/releases).
+
+### GitHub Stars
+
+According to [GitHub Star History](https://star-history.com/#withastro/astro&Date), Astro's GitHub stars were around 37,000 in early 2024 but increased to around 48,000 by the end of the year. Although that's a bit slower than 2023's increase of 15,000, it's still a jump of over 10,000 for the year. It's fair to say Astro's star count is still on a healthy upward trend.
+
+<BlogContentImage src={starHistory} alt="A chart showing the increase of GitHub stars since the project's beginning (with zero stars) in 2021. The line tends up and to the right mostly linearly, with a slight jump in late 2022." />
+
+
+### NPM Downloads
+
+Based on [npm trends](https://npmtrends.com/astro), weekly downloads of Astro were 185,902 at the start of 2024, climbing to 364,201 as of December 15. That's approximately double, so Astro is clearly growing at a steady pace here as well.
+
+<BlogContentImage src={npmTrends} alt="" />
+
+### State of JavaScript
+
+In the Meta Frameworks category of [State of JavaScript 2024](https://2024.stateofjs.com/) published in December, **Astro ranked #1 in**:
+
+- **Interest** (the proportion of positive sentiment among respondents having heard about an item)
+- **Retention** (the proportion of positive sentiment among respondents having used an item)
+- **Positivity** (the proportion of positive sentiment among respondents who expressed a sentiment)
+
+On top of that, its Usage (the proportion of respondents having used an item) rose from 4th to 2nd this year, trailing only Next.js:
```

Comment:
Microscopic detail, but given the categories are in bold text in the list and the somewhat unusual capitalization mid sentence, what do we think about also bolding this category here?

```suggestion
On top of that, its **Usage** (the proportion of respondents having used an item) rose from 4th to 2nd this year, trailing only Next.js:
```